src/_includes/layouts/installation.njk

```diff
@@ -0,0 +1,63 @@
+---
+layout: "layouts/base.njk"
+---
+
+{{ content | safe }}
+
+<div class="mt-550 pt-550 pb-450 bg-full-width bg-light">
+  <h2 class="mb-400">1. Choose an option that works for you</h2>
+  <p class="mb-450">The GCDS-component library works in multiple frameworks and can be used framework free. Choose the environment you're building in and follow the instructions to get started.</p>
+  <gcds-select id="select-environment" label="Environment you're working in" hint="Select the option that most applies" default-value="Choose an option">
```

Comment:
Can you a `select-id` attribute? The label needs it to properly label the select element.

Reply:
Oh dang, good catch!